sim/core/database_load.go

```diff
@@ -42,5 +43,13 @@ func init() {
 		}
 	}
 
+	for i, suffix := range db.RandomSuffixes {
+		simDB.RandomSuffixes[i] = &proto.ItemRandomSuffix{
+			Id:    suffix.Id,
+			Name:  suffix.Name,
+			Stats: suffix.Stats,
+		}
+	}
```

Comment:
Suffixes needed to be loaded into the DB otherwise it would crash when using a suffix item in a test.